web/templates/includes/password-options.php

```diff
@@ -0,0 +1,31 @@
+<button type="button" title="<?= _("Toggle password options") ?>" class="u-unstyled-button u-ml5 js-toggle-generate-password">
+    <i class="fas fa-caret-down icon-green"></i>
+</button>
+<div class="u-pos-relative u-mb10 u-mt10 password-options u-hidden panel">
+    <div class="u-mb20"><?= _("Select the length and characters to use when generating a password:") ?></div>
+    <div class="u-mb20">
+        <label for="v_password_length" class="form-label">
+            <?= _("Length") ?>
+        </label>
+        <input type="number" class="form-control js-password-length" name="v_password_length" id="v_password_length" min="8" value="16">
```

Comment:
- When typing it still accepts values lower then 8 chars

Reply:
Yes but the password cannot be used for the password validation rule.